addon/src/modifiers/inputmask.js

```diff
@@ -1,39 +1,49 @@
 /* globals Inputmask */
 import Modifier from 'ember-modifier';
 import { getOwner } from '@ember/application';
+import InputMask from "inputmask";
```

Comment:
Could you please run `yarn:lint:fix` inside the projects?

Reply:
In all my personal projects I use lint-staged to always run lint and prettier when I commit, so it just happens. I always forget this when doing external projects.